CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## Unreleased
+
+- Add new form 'melding interne beslissing tot samenvoeging', bump services [DL-6361]
+
+### Deploy Notes
+
+```
+drc restart migrations && drc logs -ft --tail=200 migrations
+drc up -d enrich-submission
+drc up -d submissions-dispatcher`
```

Comment:
Do we also need healing for this? Like https://github.com/lblod/app-worship-decisions-database/pull/98/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR115-R118